nomos-services/data-availability/network/src/backends/libp2p/validator.rs

```diff
@@ -75,13 +77,14 @@ pub struct DaNetworkValidatorBackend<Membership> {
     _membership: PhantomData<Membership>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct DaNetworkValidatorBackendSettings<Membership> {
-    /// Identification key
-    key: Keypair,
+    // Identification Secp256k1 private key in Hex format (`0x123...abc`). Default random.
+    #[serde(with = "secret_key_serde", default = "secp256k1::SecretKey::generate")]
+    pub node_key: secp256k1::SecretKey,
```

Comment:
I thought we would use `ed2xxxx`. There is no specification for this though. But it is important as it is attached to the `PeerId` itself.

Reply:
Updated